src/accessibility.md.njk

```diff
@@ -40,22 +40,16 @@ GDS is committed to making its websites accessible, in accordance with the Publi
 
 ### Compliance status
 
-The Design System website at [https://design-system.service.gov.uk/](/) is partially compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard, due to the non-compliances listed below.
+The Design System website at [https://design-system.service.gov.uk/](/) is fully compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard.
 
-The GOV.UK Frontend documentation website at [http://frontend.design-system.service.gov.uk/](https://frontend.design-system.service.gov.uk/) is partially compliant with the WCAG 2.1 AA standard. We know some parts of this website are not fully accessible as there are issues caused by our Technical Documentation Template.
+The GOV.UK Frontend documentation website at [http://frontend.design-system.service.gov.uk/](https://frontend.design-system.service.gov.uk/) is partially compliant with the WCAG 2.1 AA standard. We know some parts of this website are not fully accessible.
```

Comment:
Is this change meant to be in this PR?

Reply:
@lfdebrux Yeah, the reason I suggested deleting "as there are issues caused by our Technical Documentation Template" is because line 53 also mentions the template. Thought we could do without the repetition.

Reply:
Thanks, @lfdebrux! I might leave it. I've updated the PR to mention this change, too.